test/test_sparse_csr.py

```diff
@@ -3468,6 +3457,35 @@ def test_triton_bsr_dense_bmm_error_messages(self, device, dtype):
             out = torch.rand(32, 32, 2, dtype=dtype, device=device).transpose(0, -1)
             bsr_dense_mm(lhs, rhs, out=out)
 
+    @onlyCUDA
+    def test_no_triton_on_import(self):
```

Comment:
As this is not specific to sparse, do you mind moving it to say `test_cuda` (and make sure to pass _device, so it will not be filtered out?)